asset-transfer-basic/README.md

```diff
@@ -0,0 +1,142 @@
+# Asset Transfer Basic javascript Sample
+
+The asset transfer basic sample demonstrates submitting transactions
+and queries.
+
+
+## About the Sample
+
+This javascript sample application is able to use any of the asset transfer basic
+chaincode samples. It will show how to enroll an admin identity. The admin will then
+register and enroll a user identity to work with the Fabric network.
+The application will demostrate the basic operations of create, update, delete,
+and query on an asset.
+
+
+## Running the sample
+
+Use the Fabric test network utility (`network.sh`) to start a Fabric network and deploy one
+one of the sample chaincodes.
+
+Follow these step in order.
+- Start the test network with a Certificate Authority and create a channel
+```
+cd test-network
+./network.sh up createChannel -c mychannel -ca
+```
+
+- Deploy the chaincode (smart contract)
+```
+# to deploy the javascript version
+./network.sh deployCC -ccs 1 -ccv 1 -ccep "OR('Org1MSP.peer','Org2MSP.peer')" -ccl javascript -ccp ./../asset-transfer-basic/chaincode-javascript/ -ccn basic
+```
+
+- Run the application
+```
+cd asset-transfer-basic/application-javascript
+npm install
+node app.js
+```
+
+If you see an error:
+```
+    2020-08-07T20:23:17.590Z - error: [DiscoveryService]: send[mychannel] - Channel:mychannel received discovery error:access denied
+    ******** FAILED to run the application: Error: DiscoveryService: mychannel error: access denied
+```
+   or
+```
+   Failed to register user : Error: fabric-ca request register failed with errors [[ { code: 20, message: 'Authentication failure' } ]]
+   ******** FAILED to run the application: Error: Identity not found in wallet: appUser
+```
+Delete the `/fabric-samples/asset-transfer-basic/application-javascript/wallet` directory
+and retry this application.
+
+The certificate authority must have been restarted and the saved certificates for the
+admin and application user are not valid. Deleting the wallet store will force these to be reset
+with the new certificate authority.
+
+To see the SDK workings, try setting the logging to show on the console before running
+```
+export HFC_LOGGING='{"debug":"console"}'
+```
+or log to a file
+```
+export HFC_LOGGING='{"debug":"./debug.log"}'
+```
+
+## Clean up
+When you are finished, you can bring down the test network.
+The command will remove all the nodes of the test network, and delete any
+ledger data that you created:
+
+```
+./network.sh down
+```
+Be sure to delete the wallet directory create by the application.
+The stored identities will no longer be valid when restarting the network.
+
+## Configuring and running a Hardware Security Module
+
```

Comment:
I would split this readme into two, and quickly restate the commands above so that folks are sure when to complete the steps below.